### ISSUE TITLE
Terokkar Forest (Towers)

### ISSUE THREAD

**Issue by alexandru58** [dalaranwow/dalaran-wow#5744]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Every 6 hours the towers will not reset back to 0. What those suppose to mean? When a faction control all Towers and after next 6 hours, the faction that controlled the towers, they have 5 towers controlled, not 0, i mean that faction cannot control them back..

**Expected behaviour**: Tell us what should happen instead
Every 6 hours, all 5 spirit towers will become uncontrolled and capturable by standing near the tower until the bar fills in the direction of your side. If there are more players on your side it will move faster.

**Steps to reproduce the problem**:

1. Every 6 hours the towers will not reset back to 0.
2. When a faction control all Towers and after next 6 hours, the faction that controlled the towers, they have 5 towers controlled, not 0, i mean that faction cannot control them back..
3. Every 6 hours, all 5 spirit towers will become uncontrolled and capturable by standing near the tower until the bar fills in the direction of your side. If there are more players on your side it will move faster.

**Include proofs for this behaviour**

I was unable to make screenshots, but i can give you this link to check out:
https://wow.gamepedia.com/Spirit_Tower

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
#5275 5275
